src/app/components/home/lobby/lobby.component.html

```diff
@@ -25,33 +26,54 @@ <h4 class="mb-3">Where do you want to drink ?</h4>
           </div>
         </div>
 
-        <div class="col-md-9 mb-3">
-          <div class="card transparent">
-            <div class="card-body">
-              <div class="mb-3">
-                <button class="btn lobby-btn mr-3" id="" (click)="createGame()"
-                  [disabled]="activeLobby !== undefined">{{translation.text.gamesListCreate}}</button>
-                <span *ngIf="activeLobby !== undefined" class="blue-text"><i
-                    class="fas fa-exclamation-triangle"></i><span>You
-                    are
-                    in a lobby and won't be able to host a new game.</span></span>
-                <button class="btn lobby-btn mr-3" (click)="refetchGameRooms()"><i class="fas fa-redo"></i></button>
+        <div class="col-md-9">
+          <div class="row">
+
+            <div class="col-md-12 mb-3">
+              <div class="card transparent">
+                <div class="card-body">
+                  <h4><i class="fas fa-info-circle"></i> Disclaimer:</h4>
+                  <p class="text-justify">
+                    Das Spiel enthält unter anderem taktlose Witze und schwarzen Humor. Diese(r) sollte nicht zu ernst
+                    genommen werden und ist nicht gedacht um einzelne Personen oder Personen-Gruppen zu beleidigen. Sollten dir
+                    Inhalte extrem negativ auffallen, <a href="/contact">melde dich bei uns</a> und wir schauen
+                    uns dein Problem gerne an.
+                  </p>
```

Comment:
Komma und etwas umgeschrieben.

```suggestion
                  <p class="text-justify">
                    Das Spiel enthält unter anderem taktlose Witze und schwarzen Humor. Diese(r) sollte nicht zu ernst
                    genommen werden und ist nicht gedacht, um einzelne Personen oder Personen-Gruppen zu beleidigen. Sollten dir dennoch sehr extrem Inhalte auffallen, <a href="/contact">melde dich bei uns</a> und wir schauen
                    uns dein Problem gerne an.
                  </p>
```